src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -128,7 +128,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_do_put(const void *origin_addr,
                                               MPI_Aint target_disp,
                                               int target_count,
                                               MPI_Datatype target_datatype,
-                                              MPIR_Win * win, MPIDI_av_entry_t * addr,
+                                              MPIR_Win * win, MPIDI_av_entry_t * av,
```

Comment:
I like the renaming, `av` is better than the ambiguous `addr`. But I think we should do this across ch4, not just rma, for consistency reason.

Reply:
Oh, I didn't notice there are so many `MPIDI_av_entry_t *addr` in other places too. Agreed that we should replace all of them.

Reply:
Since this is a non-essential refactor and the changes will be all over the place, maybe we should delay it after the 3.4 release?

Reply:
I need get the other commits in as soon as possible for my later patches for the OpenSHMEM work. But we can delay the av renaming commit after 3.4. I will remove it from this PR for now.